requirements.txt

```diff
@@ -1,5 +1,6 @@
 -e git+https://github.com/tejado/pgoapi.git#egg=pgoapi
 geopy==1.11.0
+distribute==0.7.3
```

Comment:
trying to get travis build working, distribute==0.0 (req protobuffer) not available on travis